packages/react-components/react-infobutton/src/stories/Infobutton/InfoButtonDefault.stories.tsx

```diff
@@ -1,4 +1,15 @@
 import * as React from 'react';
 import { InfoButton, InfoButtonProps } from '@fluentui/react-infobutton';
+import { Link } from '@fluentui/react-components';
 
-export const Default = (props: Partial<InfoButtonProps>) => <InfoButton {...props} />;
+export const Default = (props: Partial<InfoButtonProps>) => (
+  <InfoButton
+    {...props}
+    content={
+      <>
+        Popover above-start lorem ipsum dolor sit amet consectetur.{' '}
```

Comment:
nit: Make this have meaningful text. E.g.:

```suggestion
        This is example content for an InfoButton.{' '}
```